utilities/stress_test.py

```
@@ -1,499 +1,399 @@
+    name: str
+    queue: Any
+    kind: str
+
+
+@dataclass
+class PipelineContext:
+    detection_frame_name: Optional[str] = None
+    labels: list[str] = field(default_factory=list)
+
+
+def on_exit(_sig: int, _frame: Any) -> None:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_frame`

(ANN401)